docker/ci/test_migrations.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.9"
-
```

Comment:
Again not strictly necessary, just resolves a build warning about `version` being deprecated